shap/explainers/_deep/deep_pytorch.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, model, data):
         self.interim = False
         self.interim_inputs_shape = None
         self.expected_value = None  # to keep the DeepExplainer base happy
-        if type(model) == tuple:
+        if isinstance(model, tuple):
```

Comment:
this is an E721 fix? If so I am missing this error code in E721 to be ignored previously or why is this fix in here?

Reply:
I believe the behaviour of this rule has changed slightly in ruff 5.0:

> [type-comparison](https://docs.astral.sh/ruff/rules/type-comparison/) (E721) now allows idioms such as type(x) is int